gallery/visualtests/pageLayouts.js

```diff
@@ -112,8 +112,7 @@ describe('Page Layout', function() {
       });
 
       it('passes a11y checks', async function() {
-        // This rule appears to not work correctly on elements that are partially scrolled off the page
-        await a11yTest(builder => builder.disableRules('color-contrast'));
+        await a11yTest(null, true);
```

Comment:
As we discussed on slack, this syntax has actually been wrong all along and these tests essentially aren't doing anything.  I think that's out of scope for this ticket, since I think it affects other places not already being edited here, and may also result in new a11y errors to fix, but did you put in a separate ticket to fix it?

Reply:
Yes, I have created a separate ticket for it. https://issues.sonatype.org/browse/RSC-1134